vpc/network/vpc.tf

```diff
@@ -0,0 +1,46 @@
+# ------------------------------------------------------------------------------
+# building vpc and all required subnets, NAT, internet gateway
+# ------------------------------------------------------------------------------
+resource "aws_vpc" "vpc" {
+    cidr_block           = var.vpc_cidr
+    enable_dns_hostnames = true
+
+    tags = {
+        Name        = "${var.environment}_vpc"
```

Comment:
I think we should discuss a naming convention for VPCs so we don't cause confusion